src/components/Order.js

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from "react";
+import RestaurantButton from "./RestaurantButton";
+
+const Order = props => {
+  const [orders, setOrders] = useState(0);
+  function orderOne() {
+    setOrders(order => orders + 1);
```

Comment:
:exclamation: Careful here - you've put `order => orders + 1`, but I think you meant:
```suggestion
    setOrders(order => order + 1);
```

Reply:
The arrow function has a parameter `order` and needs to return `order + 1` - you were referring to the state variable `orders`